app/src/main/java/org/mozilla/fenix/onboarding/DefaultBrowserNotificationWorker.kt

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.onboarding
+
+import android.app.Notification
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import androidx.core.content.ContextCompat
+import androidx.work.Worker
+import androidx.work.WorkerParameters
+import mozilla.components.support.base.ids.SharedIdsHelper
+import org.mozilla.fenix.HomeActivity
+import org.mozilla.fenix.R
+import org.mozilla.fenix.utils.Settings
+
+class DefaultBrowserNotificationWorker(
+    context: Context,
+    workerParameters: WorkerParameters
+) : Worker(context, workerParameters) {
+
+    override fun doWork(): Result {
+        ensureChannelExists()
+        NotificationManagerCompat.from(applicationContext)
+            .notify(NOTIFICATION_TAG, NOTIFICATION_ID, buildNotification())
+        return Result.success()
+    }
+
+    /**
+     * Build the default browser notification.
+     */
+    private fun buildNotification(): Notification {
+        val channelId = ensureChannelExists()
+        val intent = Intent(applicationContext, HomeActivity::class.java)
+        intent.putExtra(INTENT_DEFAULT_BROWSER_NOTIFICATION, true)
+
+        val pendingIntent = PendingIntent.getActivity(
+            applicationContext,
+            SharedIdsHelper.getNextIdForTag(applicationContext, NOTIFICATION_PENDING_INTENT_TAG),
+            intent,
+            0
+        )
+
+        with(applicationContext) {
+            val appName = getString(R.string.app_name)
+            return NotificationCompat.Builder(this, channelId)
+                .setSmallIcon(R.drawable.ic_status_logo)
+                .setContentTitle(
+                    applicationContext.getString(R.string.notification_default_browser_text, appName))
+                .setBadgeIconType(NotificationCompat.BADGE_ICON_SMALL)
+                .setColor(ContextCompat.getColor(this, R.color.primary_text_light_theme))
+                .setPriority(NotificationCompat.PRIORITY_DEFAULT)
+                .setShowWhen(false)
```

Comment:
I kept it to false since we're not really showing a notification that is time sensitive.  I'm not sure which is better but personally I like the look of the notification without the timestamp.